src/components/common/chat-popup.vue

```diff
@@ -43,52 +44,61 @@
         </template>
         <span>Click here to chat live with Helpdesk staff about Name Requests.</span>
       </v-tooltip>
-
-      <v-tooltip>
-        <v-btn x-large outlined class="chat-button" color="bcgovblue" type="submit">
-          <v-icon class="mr-2 ml-n2">mdi-forum-outline</v-icon>
-          <span class="font-weight-bold">Chat</span>
-        </v-btn>
-
-      </v-tooltip>
     </form>
   </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
-import MainContainer from '@/components/new-request/main-container.vue'
+import axios from 'axios'
+
+@Component({})
+export default class ChatPopup extends Vue {
+  readonly window = window
 
-@Component({
-  components: {
-    MainContainer
+  chatStatus = null
+  chatError = false
+
+  mounted (): void {
+    axios
+      .get(window['webChatStatusUrl'])
+      .then(response => (this.chatStatus = response.data.status))
+      .catch(error => {
+        console.log(error)
```

Comment:
Do we want this to be `console.error`? 
Can't recall whats best for Sentry..

Reply:
Sentry captures errors, yes.
I'll look at this again. At a minimum, it's missing a lint bypass.

Reply:
No biggie though, merge away if you're good with it!

Reply:
Fixed.
(It did work before -- the button is disabled -- but Sentry would not have captured the error.)
Yes, I think this error should go to Sentry.